### PR TITLE
5893 deleted docs message view

### PR DESCRIPTION
**Pull request by mrinsin**
image

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/presenter/computeds/formattedMessageDetail.js [hunk 1]
```diff
@@ -54,12 +54,13 @@ export const formattedMessageDetail = (get, applicationContext) => {
 
   if (formattedMessages[0].attachments) {
     formattedMessages[0].attachments.map(attachment => {
-      attachment.showNotServed = getShowNotServedForDocument({
+      const showNotServed = getShowNotServedForDocument({
```
Comment (kkoskelin) [inline web-client/src/presenter/computeds/formattedMessageDetail.js]: Is this stylistic, or is there a functional difference I'm also overlooking? (I'm fine with it...)

Comment (rachelschneiderman) [reply web-client/src/presenter/computeds/formattedMessageDetail.js]: We had previously added another item that was being returned in `getShowNotServedForDocument` and then in a later commit removed it. Will undo the other changes to this file 😄

File: web-client/src/presenter/computeds/getShowNotServedForDocument.test.js [hunk 2]
```diff
@@ -6,119 +6,121 @@ describe('getShowNotServedForDocument', () => {
...
-      documentId,
-      draftDocuments: [{ documentId }],
+      expect(showNotServed).toEqual(false);
     });
 
-    expect(showNotServed).toEqual(false);
-  });
+    it('should return false when the document is a correspondence document', () => {
+      // Correspondence documents should not be in case.documents and also are not served
+      const correrspondenceDocumentId = applicationContext.getUniqueId();
```
Comment (kkoskelin) [inline web-client/src/presenter/computeds/getShowNotServedForDocument.test.js]: _nit_: ooh, that identifier misspells `correspondence`.  I know, not your authorship. ;)

**Review by kkoskelin — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**